wagtail/wagtailsearch/index.py

```diff
@@ -203,6 +203,8 @@ def get_value(self, obj):
         try:
             field = self.get_field(obj.__class__)
             value = field.value_from_object(obj)
+            if hasattr(self, 'search_content') and self.search_content:
```

Comment:
I think we should make this the default behaviour of file field, instead of hiding it behind a flag. As there's no previous working behaviour and I think opening the file is what people would expect?